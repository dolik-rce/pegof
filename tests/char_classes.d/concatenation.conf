input char_classes.d/concatenation.peg
optimize concat-char-classes
header never
