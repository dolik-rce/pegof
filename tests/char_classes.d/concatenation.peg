main <- A B C D E F

A <- [a-d] / [c-f]

B <- ![A-F] / ![CD]

C <- [0-5]* / [6-9]*

D <- [XY] / [Z]*

E <- [-pqr] / [stu]

F <- [^I-K] / [^M-O]
