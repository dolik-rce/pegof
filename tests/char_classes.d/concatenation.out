main <- A B C D E F

A <- [a-dc-f]

B <- ![A-FCD]

C <- [0-56-9]*

D <-
    [XY]
    / [Z]*

E <- [-pqrstu]

F <- [^I-KM-O]
